src/webui/www/public/index.html

```diff
@@ -6,6 +6,7 @@
     <title>qBittorrent QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]</title>
     <link rel="icon" type="image/png" href="images/qbittorrent32.png" />
     <link rel="icon" type="image/svg+xml" href="images/qbittorrent-tray.svg" />
+    <link rel="stylesheet" type="text/css" href="css/palette.css?v=${CACHEID}" />
```

Comment:
Why not use `@import` at the top of style.css? So you don't need to link to palette.css in every .html files.
https://developer.mozilla.org/en-US/docs/Web/CSS/@import